tests/HelpersTest.php

```diff
@@ -0,0 +1,23 @@
+<?php
+declare(strict_types=1);
+
+it('clean up zalgo text', function (string $expected, string $zalgo): void {
+    $cleared = clearZalgoText($zalgo);
+
+    expect($cleared)->toBe($expected);
+})->with([
```

Comment:
@leMaur can you add a couple of regular texts to the tests too to ensure it doesn't clean up things we don't want cleaned? e.g. ensuring that `test123` still remains `test123` and `señor` stays `señor`